browser/ui/webui/brave_rewards_internals_ui.cc

```diff
@@ -211,6 +209,52 @@ void RewardsInternalsDOMHandler::OnGetBalance(
       std::move(balance_value));
 }
 
+void RewardsInternalsDOMHandler::GetContributions(const base::ListValue *args) {
+  if (!rewards_service_) {
+    return;
+  }
+
+  rewards_service_->GetAllContributions(base::BindOnce(
+      &RewardsInternalsDOMHandler::OnGetContributions,
+      weak_ptr_factory_.GetWeakPtr()));
+}
+
+void RewardsInternalsDOMHandler::OnGetContributions(
+    const std::vector<brave_rewards::ContributionInfo>& list) {
+  if (!web_ui()->CanCallJavascript()) {
+    return;
+  }
+
+  base::ListValue contributions;
+  for (const auto & item : list) {
+    auto contribution = std::make_unique<base::DictionaryValue>();
+    contribution->SetString("id", item.contribution_id_);
+    contribution->SetDouble("amount", item.amount_);
+    contribution->SetInteger("type", item.type_);
```

Comment:
These methods are deprecated (see [here](https://cs.chromium.org/chromium/src/base/values.h?q=values.h&sq=package:chromium&g=0&l=620)) - `SetIntKey(...)` is the way to go I think.

Reply:
Good point, fixed.